src/wx/panel.cpp

```diff
@@ -48,6 +48,32 @@ GameArea::GameArea()
     Init_2xSaI(32);
 }
 
+
+static void checkOverrides(uint rom_crc32, wxString id)
+{
+    wxString crc32;
+    wxString identifier;
+    crc32.Printf(wxT("%08X"), rom_crc32);
+
+    //const char *aux = crc32.mb_str(wxConvUTF8);
+    //const char *aux2 = id.mb_str(wxConvUTF8);
+    //fprintf(stderr, "\n\n%s -- %s\n\n", aux, aux2);
+
+    if (id == wxString("AZLE") && wxString("8E91CD13") != crc32) { // Legend of Zelda, A Link to The Past ROM hacks
```

Comment:
@rkitover This was the issue 484. The index for our `vba-over.ini` are these initials only. If the user set an override, then the original game gives the save "corrupted" messages. If he does not set it, then the hack does not save. The value `8E91CD13` is from the original ROM.

Since the overrides are only for the WX port, I decided to create this function here.